tasks.py

```diff
@@ -234,43 +234,44 @@ def load_addresses(ctx, files=[]):
         logging.info("no addresses to import")
         return
 
+    output_csv = "output.csv"
+
+    options = []
     if addr_config.get("bano", {}).get("file"):
-        logging.info("importing bano addresses")
-        conf = ctx.addresses.bano
-        additional_params = _get_cli_param(conf.get("nb_threads"), "--nb-threads")
-        additional_params += _get_cli_param(conf.get("nb_shards"), "--nb-shards")
-        additional_params += _get_cli_param(conf.get("nb_replicas"), "--nb-replicas")
-        run_rust_binary(
-            ctx,
-            "mimir",
-            "bano2mimir",
-            files,
-            "--input {ctx.addresses.bano.file} \
-            --connection-string {ctx.es} \
-            {additional_params} \
-            --dataset {ctx.dataset}".format(
-                ctx=ctx, additional_params=additional_params
-            ),
-        )
+        options.append("--bano")
+        options.append(ctx.addresses.bano.file)
     if addr_config.get("oa", {}).get("file"):
-        logging.info("importing oa addresses")
-        conf = ctx.addresses.oa
-        additional_params = _get_cli_param(conf.get("nb_threads"), "--nb-threads")
-        additional_params += _get_cli_param(conf.get("nb_shards"), "--nb-shards")
-        additional_params += _get_cli_param(conf.get("nb_replicas"), "--nb-replicas")
-        # TODO take multiples oa files ?
-        run_rust_binary(
-            ctx,
-            "mimir",
-            "openaddresses2mimir",
-            files,
-            "--input {ctx.addresses.oa.file} \
-            --connection-string {ctx.es} \
-            {additional_params} \
-            --dataset {ctx.dataset}".format(
-                ctx=ctx, additional_params=additional_params
-            ),
-        )
+        options.append("--openaddress")
```

Comment:
```suggestion
        options.append("--openaddresses")
```
The typo was fixed in https://github.com/QwantResearch/addresses-importer/pull/37